File: extensions/youtube/src/main/java/app/revanced/extension/youtube/patches/OpenVideosFullscreen.java
package app.revanced.extension.youtube.patches;

import app.revanced.extension.youtube.settings.Settings;

@SuppressWarnings("unused")
public class OpenVideosFullscreen {

    /**
     * Injection point.
     */
    public static boolean openVideoFullscreenPortrait(boolean original) {
        return Settings.OPEN_VIDEOS_FULLSCREEN_PORTRAIT.get();
    }
}

File: patches/src/main/kotlin/app/revanced/patches/youtube/player/fullscreen/OpenVideosFullscreen.kt
package app.revanced.patches.youtube.player.fullscreen

import app.revanced.patcher.patch.bytecodePatch

@Suppress("unused")
@Deprecated("Renamed to openVideosFullscreenPatch", ReplaceWith("openVideosFullscreenPatch"))
val openVideosFullscreen = bytecodePatch{
    dependsOn(openVideosFullscreenPatch)
}